content/projects/gsoc/2021/project-ideas/tekton-client-plugin.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: gsocprojectidea
+title: "Tekton client plugin for Jenkins"
+goal: "Trigger Tekton Pipelines from Jenkins Jobs."
+category: Plugins
+year: 2021
+status: draft
+sig: docs
+skills:
+- Java
+- Kubernetes
+- Tekton
+mentors:
+# To be confirmed
+#- "oleg_nenashev"
+- "waveywaves"
+- "garethjevans"
+links:
+  draft: https://docs.google.com/document/d/1_mS0gLgM7oz7PanE0X_SkJ8iUo-BPVCixXMSDO7-VMk/edit#
+---
+
+=== Background
+With the movement of traditional software architecture to microservices, there has been a growth in usage of container orchestration infrastructure (Kubernetes) and tooling around it. To adapt to this there have been plugins developed for Jenkins like the kubernetes-plugin which allow this transition to be smooth for the users to keep the same CI/CD platform and migrate their workloads. 
+
+Building on Kubernetes would mean that any CI/CD tool which has been Kubernetes-native built with building on Kubernetes as it’s main idea would thrive better than Jenkins itself. Leveraging this, we can use the capability of spinning pipelines on Tekton (Serverless CI/CD for Kubernetes) through Jenkins. Using the same vocabulary and ideas to execute Jobs from Jenkins and having the workloads of the Jobs run by Tekton in Kubernetes, this plugin aims to bridge the gap between Jenkins and Kubernetes.
+
+Jenkins currently does not support Tekton, making it hard for users to use it without prior knowledge of the CI/CD. This plugin would enable users to carry their previous Jenkins knowledge to translate it to Tekton.
+
+==== Project Details
+This project idea proposes to implement a Jenkins plugin which allows users to trigger pipelines in Tekton.
+
+
+=== Links
+
+* link:https://github.com/fabric8io/kubernetes-client [Java Kubernetes Client]
+* link:https://github.com/tektoncd/pipeline [Tekton Pipelines]
+* https://github.com/jenkinsci/tekton-client-plugin [Tekton Client Plugin]
```

Comment:
Broken rendering otherwise

```suggestion
* link:https://github.com/fabric8io/kubernetes-client[Java Kubernetes Client]
* link:https://github.com/tektoncd/pipeline[Tekton Pipelines]
* https://github.com/jenkinsci/tekton-client-plugin[Tekton Client Plugin]
```